tests/runtests.pl

```diff
@@ -2751,17 +2781,22 @@ sub cleardir {
     while($file = readdir($dh)) {
         if(($file !~ /^(\.|\.\.)\z/)) {
             if(-d "$dir/$file") {
-                cleardir("$dir/$file");
-                rmdir("$dir/$file");
+                if(!cleardir("$dir/$file")) {
+                    $done = 0;
+                }
+                if(!rmdir("$dir/$file")) {
+                    $done = 0;
+                }
             }
             else {
-                unlink("$dir/$file");
+                if(!unlink("$dir/$file") && "$file" !~ /_stunnel\.log$/) {
```

Comment:
If you don't want to remove $file stunnel that check should happen before calling unlink.

Reply:
I want to remove it, but ignore failure of that.

Reply:
Addressed with comment in newest commit.